apisix/core/pubsub.lua

```diff
@@ -49,7 +49,7 @@ local function init_pb_state()
         return "failed to load pubsub protocol: " .. err
```

Comment:
Maybe we need to apply the same fix in https://github.com/apache/apisix/pull/7639#discussion_r948679096 too?

Reply:
Done